WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/IoT/AthensAppx.cs

```diff
@@ -0,0 +1,152 @@
+        /// </summary>
+        public static readonly string HeadlessStartupAppApi = "api/iot/appx/startupHeadlessApp";
+
+        /// <summary>
+        /// IoT device package activation API.
+        /// </summary>
+        public static readonly string ActivatePackageApi = "api/iot/appx/app";
+
+        /// <summary>
+        /// Gets List of apps.
+        /// </summary>
+        /// <returns>String containing the list of applications.</returns>
+        public async Task<AppsListInfo> GetAppsListInfo()
+        {
+            return await this.Get<AppsListInfo>(AppsListApi);
+        }
+
+        /// <summary>
+        /// Gets list of headless apps.
+        /// </summary>
+        /// <returns>String containing the list of headless applications.</returns>
+        public async Task<HeadlessAppsListInfo> GetHeadlessAppsListInfo()
+        {
+            return await this.Get<HeadlessAppsListInfo>(HeadlessAppsListApi);
+        }
+
+        /// <summary>
+        /// Sets selected app as the startup app.
+        /// </summary>
+        /// <returns>Task tracking completion of the REST call.</returns>
+        public async Task UpdateStartupApp(string appId)
+        {
+            await this.Post(
+                 AppsListApi,
+                string.Format("appid={0}", Utilities.Hex64Encode(appId)));
+        }
+
+        /// <summary>
+        /// Sets the selected app as the headless startup app.
+        /// </summary>
+        /// <returns>Task tracking completion of the REST call.</returns>
+        public async Task UpdateHeadlessStartupApp(string appId)
+        {
+            await this.Post(
+                 HeadlessStartupAppApi,
+                string.Format("appid={0}", Utilities.Hex64Encode(appId)));
+        }
+
+        /// <summary>
+        /// Removes the selected app from the headless startup app list.
+        /// </summary>
+        /// <returns>Task tracking completion of the REST call.</returns>
+        public async Task RemoveHeadlessStartupApp(string appId)
+        {
+            await this.Delete(
+                 HeadlessStartupAppApi,
+                string.Format("appid={0}", Utilities.Hex64Encode(appId)));
+        }
+
+        /// <summary>
+        /// Activiates the selected app package.
+        /// </summary>
+        /// <returns>Task tracking completion of the REST call.</returns>
+        public async Task ActivatePackage(string appId)
+        {
+            await this.Post(
+                 ActivatePackageApi,
+                string.Format("appid={0}", Utilities.Hex64Encode(appId)));
+        }
+        #region Data contract
+
+        /// <summary>
+        /// Application list info.
+        /// </summary>
+        [DataContract]
+        public class AppsListInfo
+        {
+            /// <summary>
+            /// Gets or sets the default application 
+            /// </summary>
+            [DataMember(Name = "DefaultApp")]
+            public string DefaultApp { get; set; }
```

Comment:
Yep, unless it needs to be written (like if the data structure is used as the request body for a POST or something) you should use a private set and update the doc :)